packages/commercetools/composables/src/useForgotPassword/index.ts

```diff
@@ -0,0 +1,27 @@
+import { Context, UseForgotPassword, useForgotPasswordFactory, UseForgotPasswordFactoryParams } from '@vue-storefront/core';
+
+const params: UseForgotPasswordFactoryParams<any, any> = {
+  resetPassword: async (context: Context, { email, customQuery }) => {
+    const response = await context.$ct.api.customerCreatePasswordResetToken(email, customQuery);
+    const token = response?.data?.customerCreatePasswordResetToken?.value;
+    const emailObject = {
```

Comment:
you are mixing two concerns here: resetting the password itself and sending an email - the question here: If I want to now integrate an external mailer how can I do that? Normally I would write a middleware extension that will listen to `customerCreatePasswordResetToken`  or I would override it - you supposed to do the same thing with console.log - an extension that will be loaded only in the demo mode, not hardcoded mock in the factory param

Reply:
Created extension. More advanced functionality for mailExtension could be added later.